app/components-react/shared/Tabs.tsx

```diff
@@ -36,11 +36,18 @@ export default function Tabs(p: ITabs) {
     },
   ];
 
-  const data = p?.data ?? dualOutputData;
+  const data = p?.tabs ? formatTabs(p.tabs) : dualOutputData;
+
+  function formatTabs(tabs: string[]): ITab[] {
+    return tabs.map((tab: string) => ({
+      label: $t(tab),
+      key: tab,
+    }));
+  }
```

Comment:
is there a reason to restructure the props and format them here instead of just doing this formatting before passing in the tabs under the `data` prop?

Reply:
This is how it was done in the vue component.